WeatherStormPower/Models/DataParsing/JsonWeatherDataParsing.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text.Json;
+
+namespace WeatherStormPower.Models.DataParsing;
```

Comment:
`DataParsing` are not just "models". Let's move this folder out the Models folder.